sessionctx/variable/statusvar.go

```diff
@@ -129,6 +131,15 @@ var tlsVersionString = map[uint16]string{
 	tls.VersionTLS13: "TLSv1.3",
 }
 
+func getAvailableTLSVersion() string {
+	res := make([]string, 0, len(tlsVersionString))
+	for _, v := range tlsVersionString {
+		res = append(res, v)
+	}
+	sort.Strings(res)
+	return strings.Join(res, ",")
+}
+
```

Comment:
So it will always be 'TLSv1,TLSv1.1,TLSv1.2,TLSv1.3' even if https://docs.pingcap.com/tidb/dev/tidb-configuration-file#tls-version is set? And TLSv1 is not enabled by default.

Reply:
What is the our expected result? At first I thought it was showing all optional `tls_version`. Or only show the tls-version configured by the configuration file?

Reply:
From https://dev.mysql.com/doc/refman/8.0/en/encrypted-connection-protocols-ciphers.html#encrypted-connection-protocol-configuration, I think it should be the final result after apply the configuartion?